<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ColoredRectangle">
     <attribute name="title">
      <string>Colored Rectangle</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ColorTexts">
       <item>
        <widget class="QLabel" name="Red">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Green">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Blue">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>51</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ColorSpinBoxes">
       <item>
        <widget class="QSpinBox" name="RedValue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="GreenValue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="BlueValue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ColorSliders">
       <item>
        <widget class="QSlider" name="RedSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="GreenSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="BlueSlider">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="RectangleToColor">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>104</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(255, 255, 255)</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FileReader">
     <attribute name="title">
      <string>File Reader</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="fileContent">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="enterFile">
         <property name="text">
          <string>Открыть файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
